torchbraid/torchbraid_app.pyx

```diff
@@ -313,6 +365,10 @@ class BraidApp:
       core = (<PyBraid_Core> self.py_core).getCore()
       braid_SetPrintLevel(core,self.print_level)
 
+  def setMinCoarse(self, mc):
+    core = (<PyBraid_Core> self.py_core).getCore()
+    braid_SetMinCoarse(core, mc)
```

Comment:
What is the mc argument here? I have my own calculation I've been using, but is braid smart about this as well?

Reply:
Braid will stop coarsening in time, if the next coarser grid will be smaller than mc.  So, if mc = 5, and your current coarse grid has 8 time points, then braid would stop, because 
8 / m < 5

Reply:
We should definitely pass this up to the layer-parallel level.